lib/fastlane/plugin/mango/helper/docker_commander.rb

```diff
@@ -23,24 +29,27 @@ def self.start_container(emulator_args:, docker_name:, docker_image:)
         output.split("\n").last
       end
 
-      def self.stop_container(container_name:)
+      def stop_container
         Actions.sh("docker stop #{container_name}") if container_name
       end
 
-      def self.delete_container(container_name:)
+      def delete_container
         Actions.sh("docker rm #{container_name}") if container_name
       end
 
-      def self.disconnect_network_bridge(container_name: container_name)
+      def disconnect_network_bridge
         Actions.sh("docker network disconnect -f bridge #{container_name}") if container_name
       rescue StandardError
         # Do nothing if the network bridge is already gone
       end
 
-      def self.docker_exec(command:, container_name:)
-        Actions.sh("docker exec -i #{container_name} bash -l -c \"#{command}\"") if container_name
+      def docker_exec(command:)
+        if container_name
```

Comment:
Why does this command raises when the other ones just silently fail if there is not container_name set?
This method should behave like the others do or we should check for a valid `container_name` in `initialize`.